tracer/test/test-applications/azure-functions/Samples.AzureFunctions.V3InProcess/AllTriggers.cs

```diff
@@ -61,6 +73,7 @@ public async Task ExitApp([TimerTrigger(AtMidnightOnFirstJan, RunOnStartup = tru
             if (doAll || triggers.Contains("http"))
             {
                 await Attempt(() => CallFunctionHttp("simple", log), log);
+                await Attempt(() => CallFunctionHttp("error", log), log);
```

Comment:
I expected the following code. Can you confirm whether the suggestion is correct or not?

```suggestion
                await Attempt(() => CallFunctionHttp("error", log), log, expectFailure: true);
```

Reply:
Right, thanks